src/univariate/continuous/normal.jl

```diff
@@ -296,7 +298,8 @@ cf(d::Normal, t::Real) = exp(im * t * d.μ - d.σ^2 / 2 * t^2)
 
 #### Sampling
 
-rand(rng::AbstractRNG, d::Normal{T}) where {T} = d.μ + d.σ * randn(rng, T)
+rand(rng::AbstractRNG, d::Normal{T}) where {T} =
+    d.μ + d.σ * randn(rng, typeof(one(T)))
```

Comment:
`typeof(one(T))`... we need internal functions making the intention (stripping the unit) clear